src/main/resources/db/schema.sql

```diff
@@ -23,10 +25,16 @@ CREATE TABLE crew_members
     last_name   VARCHAR(255),
     position    VARCHAR(255),
     birthday    DATE NOT NULL,
-    citizenship VARCHAR(255),
-    crew_id     INT  NOT NULL,
-    CONSTRAINT crew_members_crews_FK FOREIGN KEY (crew_id) REFERENCES crews (id)
+    citizenship VARCHAR(255)
 );
 
-
-
+CREATE TABLE crews_crew_members
+(
+    crews_id        BIGINT NOT NULL,
```

Comment:
I guess it'll be better:
**crew_id
crew_member_id**
make more sense in term of reading, because it's not id of table, but id of row in table, and single row in crews is crew